src/v3/src/OktaSignIn/index.ts

```diff
@@ -214,8 +218,17 @@ export default class OktaSignIn {
   // eslint-disable-next-line class-methods-use-this
   show(): void {}
 
-  // eslint-disable-next-line class-methods-use-this
-  remove(): void {}
+  remove(): void {
+    const target = typeof this.el === 'string'
+      ? document.querySelector(this.el)
+      : this.el;
+
+    if (target) {
+      render(null, target);
+
+      this.el = null;
+    }
+  }
```

Comment:
This is needed for the test `rerender` functionality to work correct since it seems preact won't remount the widget on top of itself.